modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/identity/governance/v1/IdentityGovernanceFailureTest.java

```diff
@@ -110,4 +114,31 @@ public void testSearchInvalidGovernanceConnectorProperties() throws IOException
         validateErrorResponse(response, HttpStatus.SC_BAD_REQUEST, "IDG-50012",
                 "SelfRegistration.Enble");
     }
+
+    @Test
+    public void testUpdateGovernanceConnectorsIncorrectCategoryID() throws IOException {
```

Comment:
Shall we add method comments for test methods?

Reply:
Update PR sent: https://github.com/wso2/product-is/pull/19782